Userland/Libraries/LibHTTP/Job.cpp

```diff
@@ -66,6 +68,31 @@ static Optional<ByteBuffer> handle_content_encoding(const ByteBuffer& buf, const
             dbgln("  Output size: {}", uncompressed.value().size());
         }
 
+        return uncompressed.release_value();
+    } else if (content_encoding == "br") {
+        dbgln_if(JOB_DEBUG, "Job::handle_content_encoding: buf is brotli compressed!");
+
+        // FIXME: MemoryStream is both read and write, however we only need the read part here
```

Comment:
Could you also add a FIXME for streaming these instead of doing them all at once at the end?